core/src/main/java/io/ran/MapperGenerator.java

```diff
@@ -11,14 +11,20 @@
 public class MapperGenerator {
 	public Wrapped generate(AutoMapperClassLoader classLoader, Clazz clazz) {
 		try {
+			Path path = Paths.get("/tmp/ran/"+clazz.getSimpleName()+"Mapper.class");
+			Path pathQuery = Paths.get("/tmp/ran/"+clazz.getSimpleName()+"Query.class");
+
 			MappingClassWriter visitor = new MappingClassWriter(clazz.clazz);
 			byte[] bytes = visitor.toByteArray();
+			Files.write(path, bytes);
 			CheckClassAdapter.verify(new ClassReader(bytes),false, new PrintWriter(System.out));
 
 
 			QueryClassWriter visitor2 = new QueryClassWriter(clazz.clazz);
 			byte[] bytes2 = visitor2.toByteArray();
 			CheckClassAdapter.verify(new ClassReader(bytes2),false, new PrintWriter(System.out));
+			Files.write(pathQuery, bytes2);
```

Comment:
This is for debug only, and should not be merged

Reply:
I merge this 🤦 You are right!